docs/UserGuide.md

```diff
@@ -270,6 +270,8 @@ Action     | Format, Examples
 **Find by phone number**   | `find p/KEYWORD [MORE_KEYWORDS]`<br> e.g., `find p/91234657 85432789`
 **Find by tag**   | `find t/KEYWORD [MORE_KEYWORDS]`<br> e.g., `find t/friend roommate`
 **Delete tag** | `deltag INDEX t/KEYWORD` <br> e.g. `deltag 1 t/friend`
+**Add tag** | `addtag INDEX t/KEY_WORD [MORE_KEYWORDS]` <br> e.g. `addtag 1 t/friend t/classmate`
```

Comment:
should the optional `MORE_KEYWORDS` be `[t/MORE_KEYWORDS]` or what is written here?